client/components/Chat.vue

```diff
@@ -118,6 +129,24 @@ export default {
 		hideUserVisibleError() {
 			this.$root.currentUserVisibleError = null;
 		},
+		editTopic() {
```

Comment:
This should have a check for channel type, right now you can double click in lobby (and other types) and the input field appears.